src/Microsoft.AspNetCore.Blazor/Components/ParameterCollectionExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static class ParameterCollectionExtensions
     {
         private const BindingFlags _bindablePropertyFlags = BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.IgnoreCase;
 
-        private delegate void WriteParameterAction(ref RenderTreeFrame frame, object target);
+        private delegate void WriteParameterAction(object target, object parameterValue);
```

Comment:
The explicit delegate type isn't really needed now that this isn't `ref`

Reply:
Do you mean we could replace its usages with `Action<object, object>`? If so yes, but personally I'd prefer to give it a custom name and custom parameter names for clarity. If you mean something else, or if there's some reason why it's practically better to use `Action<object, object>`, please let me know!

Reply:
Nope, no difference, but pointing out in case you missed it, but I don't mind if you prefer it.